services/security-headers/security-headers.service.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
+
+const { BaseService } = require('..')
+
+const documentation = `
+<p>
+  The <a href="https://securityheaders.com/">Security Headers</a>
+  provide a easy mechanism to analyze HTTP response headers and give information on how to deploy missing headers.
+</p>
+</p>
+  The scan result will be hidden from the public result list and follow redirects will be on too.
+<p>
+`
+
+module.exports = class SecurityHeaders extends BaseService {
+  static get category() {
+    return 'monitoring'
+  }
+
+  static get route() {
+    return {
+      base: 'security-headers',
+      pattern: ':protocol(https|http)/:host',
```

Comment:
I would suggest using a query parameter instead, like I did for the [Bugzilla service](https://github.com/badges/shields/blob/master/services/bugzilla/bugzilla.service.js). It has become our preferred approach for new badges. 😉

Reply:
Can i define an empty pattern?

Reply:
Sorry, probably wasn't very clear. In the Bugzilla service, `baseUrl` is not only used for tests: it's used in production to select the Bugzilla host you're interested in, for instance: https://img.shields.io/bugzilla/545425?baseUrl=https%3A%2F%2Fbugs.eclipse.org%2Fbugs

So what I had in mind was something along the lines of: https://img.shields.io/security-headers?domain=https%3A%2F%2Fshields.io

@paulmelnikow do you have a strong opinion on this one?

Reply:
IMO we should use a queryParam here. We decided to leverage queryParams vs. route params to take in a user-provided url input to simplify the patterns, and we should be consistent here. 

There's probably still a couple of existing services that haven't been updated yet, but for a net new service like this one we should follow the queryParam approach. 

See https://github.com/badges/shields/pull/3633#discussion_r299231749 for more info.

Reply:
@PyvesB Is it possible to create a service without `pattern`? I tried with empty `pattern` on route, and `namedParams: {}` on examples but i obtain this error: "Error: Example for SecurityHeaders at index 0 does not declare a pattern". Can you help me with this?

Reply:
That's a good question. I've added some inline suggestions you can make that should get it running again

Reply:
@calebcartwright Thanks for the suggestions, it works just fine! I already made the suggested changes.